src/quo2/core.cljs

```diff
@@ -143,6 +147,12 @@
 ;;;; BROWSER
 (def browser-input quo2.components.browser.browser-input.view/browser-input)
 
+;;;; CALENDAR
+(def calendar quo2.components.calendar.calendar.view/calendar)
+(def calendar-day quo2.components.calendar.calendar-day.view/calendar-day)
+(def calendar-month quo2.components.calendar.calendar-month.view/calendar-month)
```

Comment:
Here only 3 components should be getting exported day, year and calendar- we want tonality our quo2 component library with figma to help avoid confusion. In the designs there is no individual month component. As such there is also only annexed for preview screens for each of these components that are in figma file.
Unless I have missed something 🤔

Reply:
i removed it from quo2 preview, also from core.cljs